Automattic/jetpack
# Publicize: Fix the is_healthy logic in the classic editor.

We removed the is_healthy property from the API and changed the logic when rendering the form in the classic editor, but it was wrong. This was only discovered as the API was updated when the Jetpack Beta was deployed to WPCOM.

This PR fixes the logic.

#### Other information:


#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Have Jetpack Social/Publicize enabled
* Connect a social network
* Create a post in the classic editor
* Click 'Edit' to view the connected networks. They will be disabled and you won't be able to toggle them on.
* With this PR, that should be fixed and they will be enabled by default as usual.
